tests/foreman/ui/test_registration.py

```diff
@@ -0,0 +1,337 @@
+        .search(query={'search': f'name={rhel_contenthost.hostname}'})[0]
+        .read()
+    )
+    for interface in host.interface:
+        interface_result = target_sat.api.Interface(host=host.id).search(
+            query={'search': f'{interface.id}'}
+        )[0]
+        # more interfaces can be inside the host
+        if interface_result.identifier == iface:
+            assert interface_result.execution
+
+
+@pytest.mark.tier2
+def test_global_registration_form_populate(
+    module_org,
+    session,
+    module_ak_with_cv,
+    module_lce,
+    module_promoted_cv,
+    default_architecture,
+    default_os,
+    target_sat,
+):
+    """Host registration form should be populated automatically based on the host-group
+
+    :id: b949e010-36b8-48b8-9907-36138342c72b
+
+    :expectedresults: Some of the fields in the form should be populated based on host-group
+        e.g. activation key, operating system, life-cycle environment, host parameters for
+        remote-execution, insights setup.
+
+    :CaseLevel: Integration
+
+    :steps:
+        1. create and sync repository
+        2. create the content view and activation-key
+        3. create the host-group with activation key, operating system, host-parameters
+        4. Open the global registration form and select the same host-group
+        5. check host registration form should be populated automatically based on the host-group
+
+    :BZ: 2056469
+
+    :CaseAutomation: Automated
+    """
+    hg_name = gen_string('alpha')
+    iface = gen_string('alpha')
+    group_params = {'name': 'host_packages', 'value': constants.FAKE_0_CUSTOM_PACKAGE}
+    target_sat.api.HostGroup(
+        name=hg_name,
+        organization=[module_org],
+        lifecycle_environment=module_lce,
+        architecture=default_architecture,
+        operatingsystem=default_os,
+        content_view=module_promoted_cv,
+        group_parameters_attributes=[group_params],
+    ).create()
+    with session:
+        session.hostgroup.update(
+            hg_name,
+            {
+                'activation_keys.activation_keys': module_ak_with_cv.name,
+            },
+        )
+        cmd = session.host.get_register_command(
+            {
+                'general.host_group': hg_name,
+                'advanced.rex_interface': iface,
+                'general.insecure': True,
+            },
+            full_read=True,
+        )
+
+        assert hg_name in cmd['general']['host_group']
+        assert module_ak_with_cv.name in cmd['general']['activation_key_helper']
+        assert module_lce.name in cmd['advanced']['life_cycle_env_helper']
+        assert constants.FAKE_0_CUSTOM_PACKAGE in cmd['advanced']['install_packages_helper']
+
+
+@pytest.mark.tier3
+@pytest.mark.usefixtures('enable_capsule_for_registration')
+def test_global_re_registration_host_with_force_ignore_error_options(
+    session, module_activation_key, default_os, default_smart_proxy, rhel7_contenthost
```

Comment:
I would suggest using `rhel_contenthost` fixture instead and limit the test only to RHEL versions you want to use.

Reply:
These test cases are migrated from test_host so didn't change it. Anyway, it is running for all rhel versions.